__tests__/integration/README.md

```diff
@@ -54,13 +54,15 @@ To make it easier for you to deploy all the sample modules to a remote environme
 to build and bundle the modules and module map for you:
 
 ```bash
-npm run build:sample-modules -- --archiveBuiltArtifacts
+npm run build:sample-modules -- --archive-built-artifacts --bundle-statics-hostname=https://cdn.example.com
```

Comment:
strictly speaking `https://cdn.example.com` is more than a [hostname](https://en.wikipedia.org/wiki/Hostname); from [a diagram in Node.JS's URL documentation](https://nodejs.org/dist/latest-v10.x/docs/api/url.html#url_url_strings_and_url_objects) this appears to be an `origin`

Reply:
Resolved in latest commit.

Reply:
```suggestion
npm run build:sample-modules -- --archive-built-artifacts --bundle-statics-origin=https://cdn.example.com
```